controllers/tab/authorDashboard/AuthorDashboardTabHandler.inc.php

```diff
@@ -61,6 +61,14 @@ function fetchTab($args, $request) {
 		$submission = $this->getAuthorizedContextObject(ASSOC_TYPE_SUBMISSION);
 		$templateMgr->assign('submission', $submission);
 
+		// Check if current author can access CopyeditFilesGrid within copyedit stage
+		$canAccessCopyeditingStage = true;
+		$userAllowedStages = $this->getAuthorizedContextObject(ASSOC_TYPE_ACCESSIBLE_WORKFLOW_STAGES);
+		if (!array_key_exists(WORKFLOW_STAGE_ID_EDITING, $userAllowedStages)) {
+			$canAccessCopyeditingStage = false;
+		}
+		$templateMgr->assign('canAccessCopyediting', $canAccessCopyediting);
```

Comment:
I think this should be `$canAccessCopyeditingStage`.

Reply:
Thanks, got that fixed.